### ISSUE TITLE
Horde Flags

### ISSUE THREAD

**Issue by Imperator2002** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#377]
I really think that the diferent hordes (the first one, maybe some kind of expedition of the Horde of draenor (the one of Ner'Zul) and the new horde) should have different flags. The flag from the first horde (first 2 games) is different from the flag of the horde from wow.
And I think the flag of the horde of draenor should be something between the flag of the first horde and the new horde. Or maybe reusing the flag of the first horde.

**Comment by zumbak04** [comment]
<details>
<summary>For Reference</summary>

First Horde:
![image](https://user-images.githubusercontent.com/46384992/118179324-33850c80-b446-11eb-925c-f0d753b071b9.png)

New Horde:
![image](https://user-images.githubusercontent.com/46384992/118179364-426bbf00-b446-11eb-9c51-a37d590bb75a.png)

</details>

**Comment by ElMariuso** [comment]
I do not agree. The first flag is widely used in game, especially for PVP. There is no evidence that the flag of the First Horde and the current Horde are different.
